products/opensuse/main.pm

```diff
@@ -207,7 +207,7 @@ sub load_otherDE_tests {
         if ($de =~ /^enlightenment$/) { load_enlightenment_tests(); }
         if ($de =~ /^mate$/)          { load_mate_tests(); }
         if ($de =~ /^lxqt$/)          { load_lxqt_tests(); }
-        loadtest "x11/shutdown";
+        load_shutdown_tests;
```

Comment:
but now this part does not behave like it did in before. It might not have any effect on subsequent test modules because there are none but e.g. we should not call the whole "dhcp cleanup" stuff when we don't need it to not confuse test reviewers and save time. WDYT?

Reply:
But what is the "danger" if whole cleanup is done?

Reply:
@okurz According to the [comment from coolo](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4548#issuecomment-370171071) and further [comment from czerw](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4548#issuecomment-370259918) in the https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4548, it was designed in the way to check whether wicked is present in system or not and avoid checking openQA variables.
> I guess what we really want to check if wicked is present. How about doing this in a little longer script instead of not doing it on openSUSE?
...
-> no need for further check_vars on products

Currently, the behavior is not changed. This part of code still will be executed only in case if wicked exists in the system (e.g. it is not executed in [Tumbleweed-KDE-Live verification run](http://10.160.65.138/tests/125#step/cleanup_shutdown/7))

The only excessive 'systemctl status network' line will be executed on some tests that potentially don't require that check.

If you finding it redundant and confusing for reviewers, then sure we can add additional check for openSUSE, but this is what coolo wanted to avoid.

Reply:
The PR you mentioned unfortunately never answered my question "Your PR does not d really explain the reason or benefit. And why would it make sense to have this only for parts of our products?" so I still don't know :( Now you have to check. And no I definitely do not want openSUSE to be handled different, I never said this. I stated that it behaves different than before and we should only do this when we see a benefit in that.

Please check a standard Tumbleweed installation case which actually *has* wicked as default. IIRC the live media are special in the regard that they have NetworkManager

Reply:
@okurz, I've discussed the ticket with pcervinka and I've restored the details.

- The problem was that DHCP cleanup was implemented for all the backends and caused an issue for s380. Please see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4540 and [poo#32698](https://progress.opensuse.org/issues/32698). Because of that, the condition was added to use it only in case of qemu backends. 

- The DHCP cleanup itself was made, because if cleanup is not done, qcow image with installed system will keep wicked settings and if you use same image in multimachine tests, there will be conflicts in dhcp and dns. 

So, all the images, that will be used by the multimachine tests, require that cleanup. 

And my opinion is that the logic was not changed. We still checking if the backend is qemu and if wicked is a network manager. I guess, a test itself should not know whether that image will be used further in multimachine test or not, it should be just ready for any further usage.

In any case, I'm open for further discussion.